aurora-mysql-plugin/src/test/java/io/cdap/plugin/aurora/mysql/AuroraMysqlSinkTestRun.java

```diff
@@ -54,20 +56,59 @@
  */
 public class AuroraMysqlSinkTestRun extends AuroraMysqlPluginTestBase {
 
+  @Before
+  public void setup() throws Exception {
+    try (Connection connection = createConnection();
+         Statement stmt = connection.createStatement()) {
+      stmt.execute("TRUNCATE TABLE MY_DEST_TABLE");
+    }
+  }
+
+  @Test
+  public void testDBSinkWithInvalidFieldType() throws Exception {
+    String inputDatasetName = "input-dbsinktest-invalid-field-type";
```

Comment:
it looks like there is a lot of code duplication, can we move the duplicated code to a common class?

Reply:
Done